packages/databricks-sdk-js/examples/workspace.ts

```diff
@@ -1,4 +1,4 @@
-import {Config, WorkspaceClient} from "..";
+import {Config, WorkspaceClient} from "../src";
```

Comment:
Let's make examples it's own package. Easier for people to understand imports that way and easier to get started.